proto/storage/cluster.proto

```diff
@@ -296,7 +296,7 @@ message ClusterHealthStatus {
 
     // For sensors not having health capability, this will be filled with gRPC connection poll. Otherwise,
     // this timestamp will be updated by central pipeline when message is processed
-    google.protobuf.Timestamp last_contact                      = 5;
+    google.protobuf.Timestamp last_contact                      = 5 [(gogoproto.moretags) = "search:\"Last Contact,store\""];;
```

Comment:
What is the reason to make this searchable?

Reply:
Made that searchable so that the garbage collection task (coming soon) is able to search clusters with no contact since <x> days. The other way would be to search for all clusters with `SensorHealthStatus : UNHEALTHY` and then filter out those that have last_contact more than <x> days old. I can use the second approach if making searchable would significantly affect db storage size.

Reply:
@connorgorman We've been mindful about adding search fields and stuff to index hence my question ^. I remember a concern that Conor had in the past.

Reply:
This should be fine. Really the nested one is the main concern

Reply:
If we don't have a UI/UX requirement to make clusters searchable by this, I would avoid adding to index. But maybe at the scale of clusters this really doesn't matter.

Reply:
It's actually not nested because we save it separately then reconstruct when grabbing the cluster